x-pack/heartbeat/cmd/root.go

```diff
@@ -23,13 +24,22 @@ var RootCmd *cmd.BeatsRootCmd
 // heartbeatCfg is a callback registered via SetTransform that returns a Elastic Agent client.Unit
 // configuration generated from a raw Elastic Agent config
 func heartbeatCfg(rawIn *proto.UnitExpectedConfig, agentInfo *client.AgentInfo) ([]*reload.ConfigWithMeta, error) {
-	//grab and properly format the input streams
-	inputStreams, err := management.CreateInputsFromStreams(rawIn, "metrics", agentInfo)
+	modules, err := management.CreateInputsFromStreams(rawIn, "metrics", agentInfo)
```

Comment:
Hmm, alright, good catch.

Reply:
Re, `synthetics`, I thought the index fields had to be one of `logs` or `metrics` ?

Reply:
Maybe it should be, but the `type` part of the datastream is definitely being set to `synthetics` in some cases: https://github.com/elastic/beats/blob/fd82372d4b0f520bdafa9338537e48b9d7bb7967/heartbeat/monitors/factory.go#L296

We have to keep the existing behaviour here to avoid breaking things.

Reply:
The packetbeat and auditbeat default type still needs to be updated to from "metrics" to "logs" from what I can see.